src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -137,7 +164,9 @@ doReadRow d k = forModuleNameFix $ \mnFix ->
     (Utf8 tableNameBS) = tableName
 
     queryStmt =
-        "SELECT rowdata FROM " <> tbl tableName <> " WHERE rowkey = ? ORDER BY txid DESC LIMIT 1;"
+        "SELECT rowdata FROM " <> tbl tableName <> " WHERE rowkey = ?" <> blockLimitStmt <> " ORDER BY txid DESC LIMIT 1;"
```

Comment:
Move `queryStmt` and `blockLimitStmt` together and near the actual `blockLimitParam` call; this just makes all the interpolated variables easier to line up and read.